### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by karanaggarwal1** [tamber/tamber-node#3]
Hi
I have just started using tamber and I think I have encountered some issue. After executing <code>npm install --save tamber</code> and running the app, I get the following error:
```
Error: Cannot find module './objects/Event'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/*****/node_modules/tamber/lib/Tamber.js:3:13)
    at Module._compile (module.js:652:30)
    at Module._extensions..js (module.js:663:10)
    at Object.require.extensions.(anonymous function) [as .js] (/****/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
```
Please let me know if I am doing something wrong.

**Comment by alexi** [comment]
It appears that the `lib/objects` directory was somehow excluded from the version of the repo published to npm. We have just published [version 1.1.1](https://www.npmjs.com/package/tamber/v/1.1.1) which should resolve the issue :)

**Comment by karanaggarwal1** [comment]
Yes that did resolve the issue, thanks a lot for the prompt reply!
